node/src/test/scala/com/wavesplatform/state/diffs/InvokeScriptTransactionDiffTest.scala

```diff
@@ -1022,4 +1014,20 @@ class InvokeScriptTransactionDiffTest extends PropSpec with PropertyChecks with
         }
     }
   }
+
+  property("Default function should not accept arguments") {
```

Comment:
it is a regular callable function, so it can accept whatever amount of arguments it needs. it's only a matter of convention: when no func/args is specified, we try to invoke default()